pom.xml

```diff
@@ -0,0 +1,51 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud.tools</groupId>
+  <artifactId>gcp-java-tools-lib</artifactId>
+  <version>0.2-SNAPSHOT</version>
+
+  <name>Google Cloud Java management client library</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <appengine.target.version>1.9.32</appengine.target.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-tools-sdk</artifactId>
+      <version>${appengine.target.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
```

Comment:
We're using Preconditions and Strings, off the top of my head.

Patrick, I don't know yet how IJ imports plugins. If it just imports a jar, then I wonder if the produced plugin jar contains its dependencies.